server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfile.java

```diff
@@ -23,11 +23,13 @@
 import javax.annotation.Nullable;
 import org.apache.commons.lang.builder.ReflectionToStringBuilder;
 import org.apache.commons.lang.builder.ToStringStyle;
+import org.sonar.db.organization.OrganizationDto;
 import org.sonar.db.qualityprofile.QualityProfileDto;
 
 public class QProfile {
```

Comment:
I guess this class was introduced to cut the link and dependency with the DTO object
Seems very odd to add a DTO back as a field